src/devices/DCMotor/README.md

```diff
@@ -5,7 +5,7 @@ This is a generic class to control any DC motor.
 DC motors are controlled by simply providing voltage on the inputs (inverted voltage inverts the direction).
 
 DC motors can be controlled with 1, 2 or 3 pins.
-Please refer to the [sample](samples/README.md) to see how to connect it.
+Please refer to the [sample](./samples/Program.cs) to see how to connect it.
```

Comment:
are we gonna copy .cs files over to the docs repo? if not I'd recommed to remove the link all together or just generically say `sample` or full link

Reply:
the links are transformed into link on the repo. So like an external link and you'll arrive on the repo.